src/Essentials/src/VersionTracking/VersionTracking.shared.cs

```diff
@@ -7,98 +7,184 @@
+		/// <summary>
+		/// Gets the current version number of the app.
+		/// </summary>
 		public static string CurrentVersion
 			=> Default.CurrentVersion;
 
-		/// <include file="../../docs/Microsoft.Maui.Essentials/VersionTracking.xml" path="//Member[@MemberName='CurrentBuild']/Docs/*" />
+		/// <summary>
+		/// Gets the current build of the app.
+		/// </summary>
 		public static string CurrentBuild
 			=> Default.CurrentBuild;
 
-		/// <include file="../../docs/Microsoft.Maui.Essentials/VersionTracking.xml" path="//Member[@MemberName='PreviousVersion']/Docs/*" />
+		/// <summary>
+		/// Gets the version number for the previously run version.
+		/// </summary>
 		public static string? PreviousVersion
 			=> Default.PreviousVersion;
 
-		/// <include file="../../docs/Microsoft.Maui.Essentials/VersionTracking.xml" path="//Member[@MemberName='PreviousBuild']/Docs/*" />
+		/// <summary>
+		/// Gets the build number for the previously run version.
+		/// </summary>
 		public static string? PreviousBuild
 			=> Default.PreviousBuild;
 
-		/// <include file="../../docs/Microsoft.Maui.Essentials/VersionTracking.xml" path="//Member[@MemberName='FirstInstalledVersion']/Docs/*" />
+		/// <summary>
+		/// Gets the version number of the first version of the app that was installed on this device.
+		/// </summary>
 		public static string? FirstInstalledVersion
 			=> Default.FirstInstalledVersion;
 
-		/// <include file="../../docs/Microsoft.Maui.Essentials/VersionTracking.xml" path="//Member[@MemberName='FirstInstalledBuild']/Docs/*" />
+		/// <summary>
+		/// Gets the build number of first version of the app that was installed on this device.
+		/// </summary>
 		public static string? FirstInstalledBuild
 			=> Default.FirstInstalledBuild;
 
-		/// <include file="../../docs/Microsoft.Maui.Essentials/VersionTracking.xml" path="//Member[@MemberName='VersionHistory']/Docs/*" />
+		/// <summary>
+		/// Gets the collection of version numbers of the app that ran on this device.
+		/// </summary>
 		public static IEnumerable<string> VersionHistory
 			=> Default.VersionHistory;
 
-		/// <include file="../../docs/Microsoft.Maui.Essentials/VersionTracking.xml" path="//Member[@MemberName='BuildHistory']/Docs/*" />
+		/// <summary>
+		/// Gets the collection of build numbers of the app that ran on this device.
+		/// </summary>
 		public static IEnumerable<string> BuildHistory
 			=> Default.BuildHistory;
 
-		/// <include file="../../docs/Microsoft.Maui.Essentials/VersionTracking.xml" path="//Member[@MemberName='IsFirstLaunchForVersion']/Docs/*" />
+		/// <summary>
+		/// Determines if this is the first launch of the app for a specified version number.
+		/// </summary>
+		/// <param name="version">The version number.</param>
+		/// <returns>True if this is the first launch of the app for the specified version number; otherwise false.</returns>
```

Comment:
```suggestion
		/// <returns><see langword="true"/> if this is the first launch of the app for the specified version number; otherwise <see langword="false"/>.</returns>
```